scripts/redcap/purge_outdated.py

```diff
@@ -20,7 +20,12 @@
 import pandas as pd
 import sys
 import sibispy
+from six import string_types
 from sibispy import sibislogger as slog
+import json
+
+# Changeable UID template for logging each dataframe by row
+UID_TEMPLATE = "Wrong date - {study_id}/{redcap_event_name}/{form}"
```

Comment:
- Move this down to the actual `log_dataframe_by_row` calls to simplify understanding it in context
- Leave out spaces - e.g. `WrongDate-{study_id}/...`